### PR TITLE
Fix overflow on big tables / small screens

### PR DESCRIPTION
**Pull request by antoinemineau**
Hello,

This PR is related to a previous one : https://github.com/sonata-project/SonataAdminBundle/pull/2169, the issue is you can't scroll on the right side of a table list on small screen. With the previous PR you couldn't access the different type of exports. I added a position absolute on this list and some markup (class fix-overflow) to apply the css and avoid wrong behaviour (display is a bit broken on IE9, IE10+, Firefox, Chrome is OK).

There is still one little remaining issue with this fix, on really small screen (~ 500px) the actions on the right (export, items per page) are on top of the actions of the left.

Related issue: https://github.com/sonata-project/SonataAdminBundle/issues/3362

Here are some screenshots : 

Without the PR
big table no scroll

With the PR
big table scroll

WIth PR and opened export
opened export

Issue on small device (500px)
overflow right buttons

Issue on IE9 with absolute positioning
ie9

Could add some IE9 specific css to fix this if needed.

### CODE REVIEW

**Comment by clytemnestra** [comment]
Hm, I see a bit of a problem having to scroll past all of the elements until down to be able to scroll. The scroll bar should be visible at all time on the screen.

File: Resources/public/css/styles.css [hunk 1]
```diff
@@ -445,3 +445,15 @@ div.sonata-filters-box div.form-group span.input-group-addon {
...
+.box .box-body {
+    overflow-x: auto;
+    overflow-y: hidden;
+}
+
+.sonata-ba-list .pull-right.fix-overflow {
+    position: absolute;
+    right: 10px;
+    bottom: 10px;
+}
```
Comment (Soullivaneuh) [inline Resources/public/css/styles.css]: New line needed.

**Comment by antoinemineau** [comment]
@Soullivaneuh What do you mean by new line needed ?

@clytemnestra You could still scroll on the side with your mouse, even if you don't see the scrollbar, I don't see how you can implement this.

**Comment by Soullivaneuh** [comment]
> @Soullivaneuh What do you mean by new line needed ?

I mean your file does not end with a new line. See the diff.

**Comment by antoinemineau** [comment]
I guess that's fix now, I have added an empty line at the end of the file.

**Comment by OskarStark** [comment]
@Soullivaneuh @antoinemineau looks like this PR should be merged on ``2.3`` what do you think?

**Comment by Soullivaneuh** [comment]
If the issue is here on 2.3 too, yes. :+1:

**Comment by core23** [comment]
Please provide a new PR for 2.3. If needed

**Pull request closed without merging**